CMakeLists.txt

```diff
@@ -345,3 +345,5 @@ if(0) # debug built library and fluid targets
   endforeach()
   message(STATUS "---------------------------------------------------------")
 endif()
+
```

Comment:
Adding these two empty lines should be reverted. This file shouldn't be changed at all